Source/ACE/Managers/GuidManager.cs

```diff
@@ -14,25 +14,33 @@ public static class GuidManager
         // players
         private const uint playerMin = 0x50000001;
         private const uint playerMax = 0x59999999;
-        private static uint player = 0x50000001;
+        private static uint player   = 0x50000001;
+
+        // admins
+        private const uint adminMin = 0x80000001;
+        private const uint adminMax = 0x89999999;
```

Comment:
0x89999999 is an arbitrary number.  Did you mean 0x8FFFFFFF, or perhaps 0x88000000?